keeper-gradle-plugin/src/test/kotlin/com/slack/keeper/KeeperFunctionalTest.kt

```diff
@@ -241,7 +248,7 @@ class KeeperFunctionalTest(private val minifierType: MinifierType) {
         .withProjectDir(projectDir)
         // TODO eventually test with configuration caching enabled
         // https://docs.gradle.org/nightly/userguide/configuration_cache.html#testkit
-        .withArguments("--stacktrace", *minifierType.gradleArgs, *args)
+        .withArguments("--stacktrace", "-Pandroid.enableR8=${minifierType.isProguard.not()}", *args)
```

Comment:
Nit but I'm not a fan of using `not()` as it just kind of makes it more verbose

```suggestion
        .withArguments("--stacktrace", "-Pandroid.enableR8=${!minifierType.isProguard}", *args)
```